crates/uv/src/windows_exception.rs

```diff
@@ -0,0 +1,127 @@
+//! Helper for setting up Windows exception handling.
+//!
+//! Recent versions of Windows seem to no longer show dialog boxes on access violations
+//! (segfaults) or similar errors. The user experience is that the command exits with
+//! the exception code as its exit status and no visible output. In order to see these
+//! errors both in the field and in CI, we need to install our own exception handler.
+//!
+//! This is a relatively simple exception handler that leans on Rust's own backtrace
+//! implementation and also displays some minimal information from the exception itself.
+
+#![allow(unsafe_code)]
```

Comment:
```suggestion
#![allow(unsafe_code)]
#![allow(clippy::print_stderr)]
```

Will help pass clippy failing CI job for windows